### ISSUE TITLE
Feature request : Add FakeAppUpdateManager.

### ISSUE THREAD

**Issue by kishanhitk** [feilfeilundfeil/flutter_in_app_update#52]
First of all, thank you very much @jonasbark for this amazing package. 
I have one feature request. Currently, to test in_app_update implementation, we have to upload the app to Internal Sharing in Play Store, so we can't test it in real-time while coding.  So, it becomes a little bit difficult for novice developers like me. Google has released [FakeAppUpdateManager](https://developer.android.com/reference/com/google/android/play/core/appupdate/testing/FakeAppUpdateManager) for Android to test InAppUpdate locally while coding. So, is it possible to add this feature to this package also?

**Comment by GabrieleOlmi** [comment]
essential feature!!

**Comment by jonasbark** [comment]
I will have a think about this. You can only *really* properly test it by using the internal sharing method in the Play Store, which is also mentioned in the official docs.

**Comment by NamanShergill** [comment]
> I will have a think about this. You can only _really_ properly test it by using the internal sharing method in the Play Store, which is also mentioned in the official docs.

Yeah, but if Google deemed it necessary to add the FakeUpdateManager, I think it only makes sense the plugin implementing the API provides it too.
